delfin/coordination.py

```diff
@@ -320,3 +322,35 @@ def _get_redis_backend_url():
             .format(backend_type=CONF.coordination.backend_type,
                     server=CONF.coordination.backend_server)
     return backend_url
+
+
+def on_node_join(event):
+    print('join watcher, group is %s, member is %s' % (event.group_id, event.member_id))
+
+
+def on_node_leave(event):
+    print('leave watcher, group is %s, member is %s' % (event.group_id, event.member_id))
+
+
+class ConsistentHashing(Coordinator):
+    GROUP_NAME = 'partitioner_group'
+
+    def __init__(self):
+        super(ConsistentHashing, self). \
+            __init__(agent_id=CONF.host, prefix="")
+
+    def join_group(self):
+        try:
+            self.coordinator.join_partitioned_group(self.GROUP_NAME)
```

Comment:
Incase of normal join_group, create group is needed first time, hope partition grp does not need creation, pls have a check